src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -151,7 +163,7 @@ private ISchema BuildSchemaFrom(GraphQLDocument document)
 
             var typeList = _types.Values.ToArray();
             typeList.Apply(schema.RegisterType);
-            schema.RegisterDirectives(directives);
+            schema.RegisterDirectives(directives.ToArray());
```

Comment:
Why? I specifically added overload to avoid the need to create an intermediate array.

Reply:
IDE was showing me an error, but that may just be a problem with VSCode.